colors/iceberg.vim

```diff
@@ -133,6 +133,10 @@ if &background == 'light'
   hi! link rubyInterpolationDelimiter String
   hi! link rubySharpBang Comment
   hi! link rubyStringDelimiter String
+  hi! link rustFuncCall Special
+  hi! link rustFuncName Title
+  hi! link rustTrait Constant
```

Comment:
`rustTrait` is linked to `rustType` by default so you can remove this line.
https://github.com/vim/vim/blob/b146e01a7e41bf5787a2b5ccc6c340ed6001f308/runtime/syntax/rust.vim#L219